### PR TITLE
Upgrade deps

### PR DESCRIPTION
**Pull request by raymondfeng**
/to @bajtos or @ritch

### CODE REVIEW

**Comment by bajtos** [comment]
I don't know what to look for, a quick superficial look did not discover any obvious problems. FWIW, LGTM.

File: package.json [hunk 1]
```diff
@@ -27,6 +27,6 @@
     "url": "https://github.com/strongloop/loopback-strorage-service/blob/master/LICENSE"
   },
   "optionalDependencies": {
-    "sl-blip": "http://blip.strongloop.com/loopback-component-storage@1.0.6"
+    "sl-blip": "http://blip.strongloop.com/loopback-component-storage@1.0.0"
```
Comment (bajtos) [inline package.json]: Is this intentional?

File: package.json [hunk 2]
```diff
@@ -7,16 +7,16 @@
...
+    "formidable": "^1.0.16"
   },
   "devDependencies": {
-    "express": "~3.4.0",
-    "loopback": "1.x.x",
-    "mocha": "~1.18.2",
-    "supertest": "~0.13.0",
-    "mkdirp": "~0.5.0"
+    "express": "^4.11.0",
+    "loopback": "2.x.x",
```
Comment (bajtos) [inline package.json]: Perhaps `"loopback": "^2.10.0"` would be better?

**Comment by raymondfeng** [comment]
@bajtos fixed.

**Comment by bajtos** [comment]
:ship:

**Pull request merged**